packages/manager/modules/pci/src/projects/project/notebooks/add/components/notebook-resources/translations/Messages_fr_FR.json

```diff
@@ -5,17 +5,17 @@
   "pci_notebook_add_resources_usecase_item_advanced": "AI Notebooks avancé (ressources GPU)",
   "pci_notebook_add_resources_usecase_item_advanced_description": "Alimenté par un maximum de quatre GPU et des douzaines de CPU. Propose de meilleures performances pour l’apprentissage deep learning et les workloads intensifs.",
   "pci_notebook_add_resources_proposed_title": "Ressources proposées",
-  "pci_notebook_add_resources_proposed_ram": "{{ ram }} RAM",
-  "pci_notebook_add_resources_proposed_storage": "{{ memory }} SSD",
-  "pci_notebook_add_resources_proposed_bandwidth": "{{ bandwidth }}/s",
+  "pci_notebook_add_resources_proposed_cpu": "CPU : {{vcores}} vCore{{vcores > 1 ? 's' : ''}}",
```

Comment:
You should not store this test into JSON file

Reply:
Changed: https://github.com/ovh/manager/pull/6676/commits/78865f1af1a30ca12d6aee2522fe671994d01da5